lib/pages/api/method/index.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
-  url: '/api/:apiMethod.html',
+  url: '/api/clearValue.html',
```

Comment:
this is an example of how one can use path parameters, you should leave the `apiMethod`.

Reply:
Reverted my change 👍